examples/CMakeLists.txt

```diff
@@ -1,19 +1,19 @@
-cmake_minimum_required(VERSION 2.8)
-
-project(DumpSlideInformation CXX)
-
-find_package(ITK REQUIRED
-  COMPONENTS
-    ITKCommon
-    ITKIOImageBase
-    ITKIOTIFF
-    IOOpenSlide
-    ITKIOJPEG
-    ITKIOPNG
-  )
-include(${ITK_USE_FILE})
-
-add_definitions(-D_USE_MATH_DEFINES=1)
-
-add_executable(DumpSlideInformation DumpSlideInformation.cxx)
-target_link_libraries(DumpSlideInformation ${ITK_LIBRARIES})
+cmake_minimum_required(VERSION 2.8)
```

Comment:
Why not cmake 3.10.2?  Or at least 

cmake_minimum_required(VERSION 2.8...3.13.2)   
See https://cmake.org/cmake/help/v3.13/command/project.html?highlight=project